app/pods/publications/index/controller.js

```diff
@@ -22,10 +22,10 @@ export default class PublicationsIndexController extends Controller {
 
   @service publicationService;
 
-  page = 0;
-  size = 25;
+  @tracked page = 0;
+  @tracked size = 10;
```

Comment:
Van @MikiDi geleerd dat er een issue is met `@tracked` query params in de Ember versie die we momenteel gebruiken. Voor query params gebruiken we dus nog de oude Ember-manier: geen `@tracked` en wijzigen via `set('page', 5)` uit `@ember/object`